libazureinit/src/media.rs

```diff
@@ -74,17 +76,32 @@ pub const PATH_MOUNT_DEVICE: &str = "/dev/sr0";
 pub const PATH_MOUNT_POINT: &str = "/run/azure-init/media/";
 
 const CDROM_VALID_FS: &[&str] = &["iso9660", "udf"];
+const MTAB_PATH: &str = "/etc/mtab";
 
-// Get a mounted device with any filesystem for CDROM
+// Public wrapper function
```

Comment:
I think we should leave the comment as-is. Although it's not currently a Rust documentation comment (three forward slashes, `///`) it is a public function and so it should have a description of what it does (and ideally how to use it).

In case you've not played with it, you can run `cargo doc` on the project which will produce a set of HTML docs for all public types. See https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html for all the details, but we should aim to have all things marked `pub` to include rustdoc comments.